### PR TITLE
Add INP to RUM browser monitoring page

### PR DESCRIPTION
**Pull request by amortemousque**
### What does this PR do? What is the motivation?

Add Interaction To Next Paint (INP) to the RUM browser monitoring page performance

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 1]
```diff
@@ -51,16 +51,17 @@ You can access performance metrics for your views in:
...
 | `view.first_byte`               | number (ns) | Time elapsed until the first byte of the view has been received.                                                                                                |
 | `view.largest_contentful_paint` | number (ns) | The moment in the page load timeline when the largest DOM object in the viewport renders and is visible on screen.                                                                                                               |
 | `view.first_input_delay`        | number (ns) | Time elapsed between a user's first interaction with the page and the browser's response.                                                                                                                                        |
+| `view.interaction_to_next_paint`| number (ns) | Time elapsed between a user's longest interaction with the page to the next paint. response.                                                                                                                                     |
```
Comment (aliciascott) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:64]:
```suggestion
| `view.interaction_to_next_paint`| number (ns) | Time elapsed between a user's longest interaction with the page to the next paint response.                                                                                                                                     |
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Just a small change to remove a period where it shouldn't be :) Please request re-review after this is fixed and I'll get it merged.

**Review by anediaz — APPROVED** [review]

**Review by aliciascott — APPROVED** [review]
looks good!

**Comment by aliciascott** [comment]
Hi @amortemousque is this ready for merge and if so do you want us to go ahead and merge it ?

**Pull request merged**